node/resolvers/search/facets.ts

```diff
@@ -0,0 +1,15 @@
+export const resolvers = {
+  Facet: {
+    values: (facet: any, args: any) => {
+      const { values } = facet
+      if (args.from === undefined) {
+        return values
+      } else {
+        return values.slice(args.from ?? 0, args.to)
```

Comment:
is the `if(args.from === undefined)` necessary since you are using `0` as fallback here?